eventGenerator/generator.js

```diff
@@ -0,0 +1,190 @@
+import fs from 'fs';
+import uuid from 'uuid';
+import resolveStorage from 'resolve-storage';
+import storageDriver from 'resolve-storage-file';
+
+import eventTypes from '../common/events/index';
+import HNServiceRest from './services/HNServiceRest';
+
+const MAX_STORY_COUNT = 50;
+
+let events = [];
+let users = {};
+let storyIds = [];
+
+const dbPath = './storage.json';
+let canceled = false;
+
+const isCanceled = () => canceled;
+
+const storage = resolveStorage({
+  driver: storageDriver({ pathToFile: dbPath })
+});
+
+const {
+  USER_CREATED,
+  STORY_CREATED,
+  STORY_UPVOTED,
+  COMMENT_CREATED
+} = eventTypes;
+
+const addEvent = (type, aggregateId, timestamp, payload) =>
+  events.push({
+    type,
+    aggregateId,
+    timestamp,
+    payload
+  });
+
+const generateUserEvents = name => {
+  const aggregateId = uuid.v4();
+  addEvent(USER_CREATED, aggregateId, new Date(3600 * 24 * 1000).getTime(), {
```

Comment:
`new Date(3600 * 24 * 1000).getTime()` === `3600 * 24 * 1000`. P.S. Extract to the const

Reply:
fixed